index.js

```diff
@@ -284,11 +285,16 @@ export async function registerSpace(opts) {
 
 /**
  * @param {string} proofPath
+ * @param {object} opts
+ * @param {string} opts.base64
  */
-export async function addSpace(proofPath) {
+export async function addSpace(proofPath, opts) {
   const client = await getClient()
-  const delegation = await readProof(proofPath)
-  const space = await client.addSpace(delegation)
+  const proof = opts.base64 
+    ? await proofFromString(opts.base64)
+    : await proofFromPath(proofPath)
+  const space = await client.addSpace(proof)
+  await client.setCurrentSpace(space.did())
```

Comment:
I reckon space add should set the current space. It was annoying to have to do that manually after calling space add.